### ISSUE TITLE
[algolia-pro] algolia-pro.css overrides bulleted lists across the site

### ISSUE THREAD

**Issue by thekenshow** [getgrav/grav-premium-issues#277]
When I insert the following into my template (per https://getgrav.org/premium/algolia-pro/docs/frontend#usage):

`{% include 'partials/algolia-pro/instantsearch.html.twig' ignore missing with { index: 'crawl' } %}`

Existing bulleted lists all over the site are overridden. Seems to be caused by the following style in `user/plugins/algolia-pro/build/css/algolia-pro.css`:

```
menu,ol,ul {
  list-style:none;
  margin:0;
  padding:0
}
``` 

There's another style earlier in `algolia-pro.css` that's limited to `[data-algolia-pro]` content:

```
[data-algolia-pro] li,[data-algolia-pro] ul {
  list-style:none;
  margin:0;
  padding:0
}
```

Maybe the unrestricted menu, ol, ul override is getting pulled in from tailwind base/components/utilities?

**Comment by w00fz** [comment]
Hey @thekenshow,

I have now fixed this to scope the `base` and `components` layers from Tailwind. The `base` particularly is what resets the browser defaults (normalizer work Tailwind does), which is what resets the lists as you reported. My patch will now keep those name-spaced within the `[data-algolia-pro]` container. 

Unfortunately, the remaining `utilities` layer that Tailwind requires cannot be scoped like the others, because depends on global class naming. If this is a problem, the next logical step would be to disable the built-in CSS and provide your own.

This fix is set for Algolia Pro v1.0.5. There are other issues open though so won't be released quite yet.

**Comment by thekenshow** [comment]
Thanks for the update.
